source/test/sgp_mode_test/functional_tests/HealthHost_Instructions.test.cc

```diff
@@ -11,110 +11,117 @@
 #include "../../../default_mode/DataNodes.h"
 #include "../../../sgp_mode/SGPDataNodes.h"
 
+/**
+ * This file is dedicated to testing the HealthHost specific instructions Donate and Steal
+ */
 
 TEST_CASE("When DONATION_STEAL_INST is 1 then Symbiont with 'Steal' instruction properly takes CPU cycles from HealthHost", "[sgp][sgp-functional]"){
- 
-  emp::Random random(1);
-  SymConfigSGP config;
-  config.SEED(0);
-  config.INTERACTION_MECHANISM(HEALTH);
-  config.SYMBIONT_TYPE(1);
-  config.MUTATION_RATE(0.0);
-  config.MUTATION_SIZE(0.00);
-  config.TRACK_PARENT_TASKS(1);
-  config.VT_TASK_MATCH(1);
-  config.HOST_ONLY_FIRST_TASK_CREDIT(1);
-  config.SYM_ONLY_FIRST_TASK_CREDIT(1);
-  config.HOST_REPRO_RES(10000);
-  config.DONATION_STEAL_INST(1);
-  config.CPU_TRANSFER_CHANCE(1);
-
-
-
-  SGPWorld world(random, &config, LogicTasks);
-
-
-  //Creates a host that only does NAND operations
-  emp::Ptr<HealthHost> host = emp::NewPtr<HealthHost>(&random, &world, &config, CreateNandProgram(100));
-  //Creates a symbiont that only does Nand operations
-  emp::Ptr<SGPSymbiont> sym = emp::NewPtr<SGPSymbiont>(&random, &world, &config, CreateParasiteNandProgram(100,94));
-
-  //Adds host to world and sym to host.
-  world.AddOrgAt(host, 0);
-  host->AddSymbiont(sym);
-    
-  WHEN("A symbiont performs a Steal instruction"){
-    sym->GetCPU().RunCPUStep(0, 100);
-    (*(sym->GetCPU().state.tasks_performed))[1] = 0;
-    THEN("The host should be set to lose 4 cycles to the symbiont"){
-      REQUIRE(host->GetCyclesGiven() == -1);
-    }
+  
+  GIVEN("An SGPWorld with mutation disabled populated with parasites"){
```

Comment:
Again, refocus the Given on what is relevant to the test, think about what is needed for it to read clearly if the test fails. Something like this sentence would be the goal:
"Given a health host and parasite that only do NAND, when the parasite runs its entire genome,  the host should lose all cycles to the parasite"